src/controller/handler/connections.ts

```diff
@@ -37,38 +36,75 @@ export class Connections implements IAgentResponseHandler {
         Route will be "topic"
         topic will be "connections"
     */
-    public async handlePost(agentUrl: string, adminApiKey: string, route: string, topic: string, body: any): Promise<any> {
+    public async handlePost(agentUrl: string, agentId: string, adminApiKey: string, route: string, topic: string, body: any): Promise<any> {
```

Comment:
agentUrl doesn't need to be passed in any more, since the method has everything to build the URL.  However, it means the part of the URL that is common is constructed in one place. thoughts?

Reply:
I think passing in the URL is fine, I think it's probably more important to centralize the URL construction logic